### ISSUE TITLE
Downgrade CI Node from v13 to v10

### ISSUE THREAD

**Issue by chefsale** [near/rainbow-bridge#120]
Currently we use node 13 on our CI infrastructure. We need to downgrade it to node 10.

It makes this PR to fail: https://github.com/near/rainbow-bridge/pull/106 due to the discussed problem with node versions above 10 in https://github.com/near/rainbow-bridge/issues/96

```
[2020-06-24T19:17:21Z] Output:
--
  | [2020-06-24T19:17:21Z] gyp info it worked if it ends with ok
  | [2020-06-24T19:17:21Z] gyp info using node-gyp@5.0.7
  | [2020-06-24T19:17:21Z] gyp info using node@13.9.0 \| linux \| x64
  | [2020-06-24T19:17:21Z] gyp info find Python using Python version 2.7.16 found at \"/usr/bin/python\"
  | [2020-06-24T19:17:21Z] gyp info spawn /usr/bin/python
  | [2020-06-24T19:17:21Z] gyp info spawn args [
  | [2020-06-24T19:17:21Z] gyp info spawn args   '/var/lib/buildkite-agent/.nvm/versions/node/v13.9.0/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
  | [2020-06-24T19:17:21Z] gyp info spawn args   'binding.gyp',
  | [2020-06-24T19:17:21Z] gyp info spawn args   '-f',
  | [2020-06-24T19:17:21Z] gyp info spawn args   'make',
  | [2020-06-24T19:17:21Z] gyp info spawn args   '-I',
  | [2020-06-24T19:17:21Z] gyp info spawn args   '/var/lib/buildkite-agent/builds/buildkite-medium-i-0949fb3118f038287-1/nearprotocol/rainbow-bridge/environment/node_modules/scrypt/build/config.gypi',
  | [2020-06-24T19:17:21Z] gyp info spawn args   '-I',
  | [2020-06-24T19:17:21Z] gyp info spawn args   '/var/lib/buildkite-agent/.nvm/versions/node/v13.9.0/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',

```

cc: @nearmax @ailisp

**Comment by chefsale** [comment]
Example of the failure: https://buildkite.com/nearprotocol/rainbow-bridge/builds/119#9207a676-5eed-411a-a84c-65180df0b882

**Comment by ailisp** [comment]
I think we should target 12 or 14, 10 is already in LTS state: https://nodejs.org/en/about/releases/

**Comment by chefsale** [comment]
As discussed in https://github.com/near/rainbow-bridge/issues/96 nothing above 10 works with ganache-core, so we will need to stick to 10 until they upgrade the package.

**Comment by chefsale** [comment]
cc: @nearmax for the ci failing

**Comment by ailisp** [comment]
I think you mention it works with 12: https://github.com/near/rainbow-bridge/issues/96#issuecomment-646604651
near-api-js only works with 12+, we should avoid 10.

And i believe, if keep ` ./scripts/start_ganache.sh & ` it still pass tests: https://buildkite.com/nearprotocol/rainbow-bridge/builds/116#b2df9518-9e49-4852-9836-4810564ff899

For docker, you don't have to enforce ganache use same base image as bridge services, that's not how docker-compose supposed to work. ganache should be launch with a separate container (we're free to use 10 or 12 in that image), and from the point of view of near bridge services, it's just a url `http://ganache:9545`, no different than url of `http://mainnet.infura.io`

**Comment by chefsale** [comment]
It doesn't work with 12, I validated that later on. It has to be 10.

**Comment by ailisp** [comment]
I still don't think it's a good idea to just downgrade to 10 for ganache-cli, using start_ganache.sh plus npm i -g ganache-cli then start manually with ganache-cli works with 12. I think we can wrap `./scripts/start_ganache.sh` as `pm2 ./scripts/start_ganache.sh`, as enviroment/index.js start ganache command, that would work in non docker case. For docker case, ganache-cli in a different image would work.

**Comment by chefsale** [comment]
I'm fine with both solution, we have been discussing this for days, as long as we just get it done with. I will leave @nearmax to decide which option is preferable here.

**Comment by nearmax** [comment]
> I still don't think it's a good idea to just downgrade to 10 for ganache-cli, using start_ganache.sh plus npm i -g ganache-cli then start manually with ganache-cli works with 12. I think we can wrap `./scripts/start_ganache.sh` as `pm2 ./scripts/start_ganache.sh`, as enviroment/index.js start ganache command, that would work in non docker case. For docker case, ganache-cli in a different image would work.

@ailisp Could you please implement it?

**Comment by ailisp** [comment]
> @ailisp Could you please implement it?

Sure, will do today

**Comment by ailisp** [comment]
Fixed in #135
